### PR TITLE
Add the mouse plugin

### PR DESCRIPTION
**Pull request by Moop1e**
Sometime useful to disable the mouse acceleration for precision works or games ;)

### CODE REVIEW

File: plugins/mouse.plugin/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+	"label": "Disabling mouse acceleration",
```
Comment (satya164) [inline plugins/mouse.plugin/metadata.json]: The label should read like `This plugin will **Disable mouse acceleration**`

File: plugins/mouse.plugin/metadata.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	"description": "Get better mouse precision.",
```
Comment (satya164) [inline plugins/mouse.plugin/metadata.json]: Can we improve the description please, it's confusing right now.

File: plugins/mouse.plugin/setup.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cat > /etc/X11/xorg.conf.d/50-mouse-acceleration.conf <<EOF
+Section "InputClass"
+	Identifier "Fedy Mouse Configuration"
```
Comment (satya164) [inline plugins/mouse.plugin/setup.sh]: This should be more descriptive. `Fedy Mouse Configuration` doesn't tell me anything about what this does.

**Comment by Moop1e** [comment]
Ok, understood. I'll fix it.

**Comment by satya164** [comment]
cc @Abhinav1217

**Pull request merged**